components/brave_rewards/resources/page/storage.ts

```diff
@@ -22,6 +22,7 @@ export const defaultState: Rewards.State = {
   donationAbilityTwitter: true,
   reconcileStamp: 0,
   ui: {
+    disconnectWalletError: false,
```

Comment:
I think we need to add a similar line to the file at:

```
components/brave_rewards/resources/android_page/storage.ts
```

to fix the Android build.

Reply:
updated 👍